systemd/generate.go

```
@@ -203,24 +211,53 @@
+		GetTimerFileDropInDir(config.Title, config.SubTitle):   collect.All(config.DropInFiles, u.IsTimerDropIn),
+		GetServiceFileDropInDir(config.Title, config.SubTitle): collect.All(config.DropInFiles, collect.Not(u.IsTimerDropIn)),
 	}
 	for dropInDir, dropInFiles := range dropIns {
 		dropInDir = filepath.Join(systemdUserDir, dropInDir)
-		if err = CreateDropIns(dropInDir, dropInFiles); err != nil {
+		if err = u.createDropIns(dropInDir, dropInFiles); err != nil {
 			return err
 		}
+		if config.UnitType == UserUnit && u.user.SudoRoot {
+			// we need to change the owner to the original account
+			afero.Walk(u.fs, dropInDir, func(path string, info fs.FileInfo, err error) error {
```

golangci-lint (1.64.8)

Error return value of `afero.Walk` is not checked

(errcheck)